roles/splunk/tasks/configure_idxc_manager.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Run splunk version command to check currently installed version
+  command: "{{ splunk_home }}/bin/splunk version --answer-yes --auto-ports --no-prompt --accept-license"
+  register: current_version
+  become: true
+  become_user: "{{ splunk_nix_user }}"
+  changed_when: false
+
+- name: Set splunk version number
```

Comment:
Move this to the version check task as well. Also, I think you should only be checking the major and minor version (X.Y).

Reply:
Good point - Makes much better sense since only version >= 8.1 has the updated clustering mode values.